charts/radar-output/templates/deployment.yaml

```diff
@@ -53,6 +53,7 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           command:
             - radar-output-restructure
+            - radar-output-restructure
```

Comment:
Is this additional command needed? I think the lint is failing because there's no version update in `radar-output`.

Reply:
Ooops, I missed this. This indeed should be removed. I failed at self-review. By bad..